src/components/Communicator/CommunicatorDialog/CommunicatorDialogBoardItem.component.js

```diff
@@ -387,6 +389,68 @@ class CommunicatorDialogBoardItem extends React.Component {
       </IconButton>
     );
 
+    const PublicBoardInfo = () => {
+      return (
+        <Dialog
+          onClose={this.handleDialogClose.bind(this)}
+          aria-labelledby="board-info-title"
+          open={this.state.openBoardInfo}
+          className="CommunicatorDialog__boardInfoDialog"
+        >
+          <DialogTitle
+            id="board-info-title"
+            onClose={this.handleDialogClose.bind(this)}
+          >
+            {board.name}
+          </DialogTitle>
+          <DialogContent>
+            <DialogContentText>
+              <Typography variant="body1" gutterBottom>
+                <b>{intl.formatMessage(messages.boardInfoName)}:</b>{' '}
+                {board.name}
+              </Typography>
+              <Typography variant="body1" gutterBottom>
+                <b>{intl.formatMessage(messages.boardInfoAuthor)}:</b>{' '}
+                {board.author}
+              </Typography>
+              <Typography variant="body1" gutterBottom>
+                <b>{intl.formatMessage(messages.boardInfoLocale)}:</b>{' '}
+                {board.locale}
```

Comment:
Show the tranlation of the language name` for the user like in this example: https://github.com/cboard-org/cboard/blob/cbfedc8bcec0e287f6ab880972dc75300bdd0734/src/components/Settings/Language/Language.component.js#L238-L239